conda_build/utils.py

```diff
@@ -1768,3 +1768,38 @@ def sha256_checksum(filename, buffersize=65536):
         for block in iter(lambda: f.read(buffersize), b''):
             sha256.update(block)
     return sha256.hexdigest()
+
+
+class GPGWrapper(object):
```

Comment:
This will be eventually be used by Conda, CB and CPH. So, this should ideally go into cph.utils